acos_client/v30/slb/server.py

```diff
@@ -23,11 +23,12 @@ class Server(base.BaseV30):
     def get(self, name, **kwargs):
         return self._get(self.url_prefix + name, **kwargs)
 
-    def create(self, name, ip_address, status=1, **kwargs):
+    def create(self, name, ip_address, status=1, conn_limit=8000000, **kwargs):
         params = {
             "server": {
                 "name": name,
                 "host": ip_address,
+                "conn-limit": conn_limit,
```

Comment:
Since the conn-limit isn't required by acos we shouldn't be sending it when it's not specified.